exercises/change/change_test.go

```diff
@@ -0,0 +1,68 @@
+package change
+
+import (
+	"reflect"
+	"testing"
+)
+
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
+	}
+}
+
+func determineExpected(expected interface{}) (valid bool, list []int) {
```

Comment:
I will attempt a move of the handling code to the generator to make the _test logic more normal.